tests/unit/services/test_config_deployer.py

```
@@ -0,0 +1,204 @@
+        # File should be overwritten with original value
+        assert local_file.read_text() == "1.75.0"
+
+    def test_deploy_role_fails_for_invalid_role(self, deployer: ConfigDeployer) -> None:
+        """Test that deploy_role fails for non-existent role."""
+        result = deployer.deploy_role("nonexistent")
+
+        assert result.success is False
+        assert "does not have a config directory" in result.message
+
+    def test_deploy_all_deploys_all_roles(
+        self, deployer: ConfigDeployer, temp_home: Path
```

Ruff (0.14.10)

Unused method argument: `temp_home`

(ARG002)